### DISCUSSION TITLE
SVGA card support

### DISCUSSION THREAD

**Discussion by darvin** [dbalsom/marty#19] [Ideas]
Which SVGA card is the best candidate for implementation?

some information:

This page is home to the [FreeVGA Project](https://www.scs.stanford.edu/10wi-cs140/pintos/specs/freevga/freevga.htm) - it doesn't have svga specification

[SVGA specification, $9.99 paywall](https://www.coursehero.com/file/33600862/Chipsetdocx/)

VMWare [reference implementation](https://github.com/prepare/vmware-svga) and [spec](https://github.com/prepare/vmware-svga/blob/master/doc/svga_interface.txt) - test Wolfstain 3D / Doom / Quake / Duke Nukem @ DOS 5.22 in VMWare playe with this implementation to prove a usefulness

**Comment by dbalsom** [comment]
I would like to keep the issues section for bug reports and problems with the emulator's current feature set. If you would like to discuss feature requests please start a thread in discussions

**Comment by dbalsom** [comment]
I need to finish the VGA implementation before worrying about Super VGA, but possibly something like the Tseng Labs ET4000

**Reply by darvin** [reply]
if you dont mind ill collect useful information for future reference in the first message

**Reply by dbalsom** [reply]
go for it
